vacs-map-app/src/components/pages/ClimateChangeOverview.vue

```diff
@@ -0,0 +1,12 @@
+<template>
+  <LayoutOverview topic-label="climate change">
+    climate change
+  </LayoutOverview>
+</template>
+
+<script setup>
+import LayoutOverview from '@/components/layouts/LayoutOverview.vue';
+</script>
+
+<style scoped>
+</style>
```

Comment:
Example of a topic overview page. Note that the top section is covered by the `LayoutOverview` component.